### PR TITLE
Improve workflow for when the API key cannot be found by the script: …

### PR DESCRIPTION
**Pull request by tatellos**
…Ask the user for the key, and tell them how to obtain one

### CODE REVIEW

**Comment by thomas-endres-tng** [comment]
Oh oh, I think we did the same thing twice ... Have a look at feat/store_api_key

**Comment by tatellos** [comment]
It's not the same feature, I saw that branch. This code works with the store_api_key feature as well.
But it's quite similar - you can remove the user-entering-the-key part from this PR if you want :)

File: please.sh [hunk 1]
```diff
@@ -136,24 +136,32 @@ get_key_from_keychain() {
...
+      debug "OPENAI_API_KEY not set and no supported keychain available."
+      exitStatus=1
       ;;
   esac
 
   if [ "${exitStatus}" -ne 0 ]; then
-    echo "OPENAI_API_KEY not set and unable to find it in keychain."
-    echo "Run please -a to store it in the keychain."
-    exit 1
+    echo "OPENAI_API_KEY not set and unable to find it in keychain. See the README on how to persist your key."
```
Comment (tatellos) [inline please.sh:154]: you can just remove the diff from here to line 164 if you don't want two so similar features

**Comment by thomas-endres-tng** [comment]
OK, you are right. I'll merge it :-)

**Pull request merged**